config/config.template.ini

```diff
@@ -494,17 +494,21 @@ SubD =
 
 poc_title = %DESCR%: %MSG%
 poc_text = %DATE% %TIME% - %DESCR%: %MSG%
+poc_ric = %DESCR%
 
 # Section for ZVEI
 # default prio for all ZVEI - except you specify it different
 zvei_prio = true
 zvei_title = Alarm: %ZVEI%
 zvei_text = %DATE% %TIME%: %ZVEI%
+# zvei_ric working at 6 characters or more
+zvei_ric = %DESCR%
 
 # Section for FMS
 fms_prio = true
 fms_title = FMS: %FMS%
 fms_text = %DATE% %TIME%: %FMS%%BR%Status: %STATUS% - Direction: %DIRT% - TSI: %TSI% %LPAR%%DESCR%%RPAR%
+fms_vehicle = %DESCR%
```

Comment:
Da es immer mehr Parameter werden - bitte kurz dokumentieren was diese Parameter eigentlich aussagen

Reply:
steht bereits unten
# %DESCR%    = Description, if description-module is used

Reply:
ich meinte nicht die Wildcards - sondern was die Config Felder an sich bedeuten.
Wo kommt das auf der anderen Seite raus, was ich da eintrage?

Reply:
ist jetzt beschriftet. Ist das so ausreichend?

Reply:
Jou - Danke